macros/metrics/base/build_in/column_default.sql

```diff
@@ -92,3 +92,31 @@
 {% macro re_data_metric_not_match_regex_percent(column_name, config) %}
     {{ percentage_formula(re_data_metric_not_match_regex(column_name, config), re_data_metric_row_count()) }}
 {% endmacro %}
+
```

Comment:
Hey, one comment for this + not_match_regex macros, currently those are added in `column_default.sql` file, and they are not really column defaults, opposed to other columns there. Could you move those newly added metrics to new file? Idea was for this one to contain only metrics which are enabled by default in re_data.

Reply:
btw for `regex` metrics as they are also optional I would also move them to optional file